cylc/flow/scripts/graph.py

```diff
@@ -215,19 +243,123 @@ def get_option_parser():
     return parser
 
 
+def dot(opts, lines):
+    """Render a graph using graphviz 'dot'.
+
+    This crudely re-parses the output of the reference output for simplicity.
+
+    This functionality will be replaced by the GUI.
+
+    """
+    if not which('dot'):
+        sys.exit('Graphviz must be installed to render graphs.')
+
+    # set filename and output format
+    if opts.output:
+        filename = opts.output
+        try:
+            fmt = filename.split('.')[1]
```

Comment:
In case someone puts more than 1 dot in the filename, should it use the first or last?
```suggestion
            fmt = filename.rsplit('.', 1)[1]
```

Reply:
I don't think `dot` accepts any formats like that but why not.